engine/src/main/scala/cromwell/webservice/metadata/IndexedJsonValue.scala

```diff
@@ -0,0 +1,79 @@
+package cromwell.webservice.metadata
+
+import spray.json._
+
+import scalaz.{Monoid, Semigroup}
+import scalaz.Scalaz._
+
+private object IndexedJsonValue {
+   implicit val IndexedJsonSemigroup: Semigroup[IndexedJsonValue] = {
```

Comment:
The naming is now inconsistent, plus I'm not sure how this is working given that it's declared as a semigroup?